src/components/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import profile from "../image/profile.jpeg";
+
+function Header() {
+  return (
+    <div className="header">
```

Comment:
```suggestion
    <header className="header">
```

해당 부분과 함께 다른 부분에서도 `main`, `aside`, `section`, `article`과 같은 **시맨틱 태그**에 대해 공부해보시면 좋을 거 같아요!

https://developer.mozilla.org/ko/docs/Glossary/Semantics